app/javascript/dashboard/featureFlags.js

```diff
@@ -38,6 +38,7 @@ export const FEATURE_FLAGS = {
   CHANNEL_INSTAGRAM: 'channel_instagram',
   CONTACT_CHATWOOT_SUPPORT_TEAM: 'contact_chatwoot_support_team',
   WHATSAPP_EMBEDDED_SIGNUP: 'whatsapp_embedded_signup',
+  CRM_V2: 'crm_v2',
```

Comment:
won't need this for now since we aren't using this flag in any frontend features, let add it when we do